### PR TITLE
Add "disable auto safe mode" option

### LINKED ISSUE #308: Update launcher to accept and use 'disable auto safe mode' option

### PR DESCRIPTION
**Pull request by Steve-Mcl**
closes #308 

## Description

Adds "disable auto safe mode" option

Adds test: 

```
  Launcher
    disable auto safe mode
      ✔ is disabled by default
      ✔ can be set by user
```

## Related Issue(s)

Owner: #308 
Parent: https://github.com/FlowFuse/flowfuse/issues/4552

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: lib/launcher.js [hunk 1]
```diff
@@ -271,7 +279,7 @@ class Launcher {
                 Object.assign(data, body)
             }
         }
-        return got.post(this.options.forgeURL + '/logging/' + this.options.project + '/audit', {
+        return got.default.get.post(this.options.forgeURL + '/logging/' + this.options.project + '/audit', {
```
Comment (hardillb) [inline lib/launcher.js:282]:
```
        return got.default.get.post(this.options.forgeURL + '/logging/' + this.options.project + '/audit', {
```

I'm getting

```
Failed to poll NR on  http://127.0.0.1:12081/ TypeError: got.default.get.head is not a function
    at statusPoll (/opt/share/projects/flowforge/dev-env/packages/nr-launcher/lib/launcher.js:441:51)
    at Timeout._onTimeout (/opt/share/projects/flowforge/dev-env/packages/nr-launcher/lib/launcher.js:512:17)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
```

File: lib/launcher.js [hunk 2]
```diff
@@ -430,7 +438,7 @@ class Launcher {
                     opts.throwHttpErrors = false
                 }
                 // Use a HEAD request to minimise data transfer
-                const res = await got.head(pollUrl, opts)
+                const res = await got.default.get.head(pollUrl, opts)
```
Comment (hardillb) [inline lib/launcher.js:441]:
```
Failed to poll NR on  http://127.0.0.1:12081/ TypeError: got.default.get.head is not a function
    at statusPoll (/opt/share/projects/flowforge/dev-env/packages/nr-launcher/lib/launcher.js:441:51)
    at Timeout._onTimeout (/opt/share/projects/flowforge/dev-env/packages/nr-launcher/lib/launcher.js:512:17)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
```

Comment (hardillb) [inline lib/launcher.js:441]:
```suggestion
                const res = await got.default.head(pollUrl, opts)
```

**Review by hardillb — CHANGES_REQUESTED** [review]
Why did we need the changes to the `got` calls?

File: lib/launcher.js (hunk 2 above)
Comment (Steve-Mcl) [reply lib/launcher.js]: `yeah, seems I had multiple cursors active when I modified the code. I corrected the `got.default.get` (and missed the fact I'd messed up the others.

Pushed an update to revert the 2 places that should not have been modified

**Comment by Steve-Mcl** [comment]
> Why did we need the changes to the `got` calls?

Only the `got.get` was supposed to be updated since it is mocked in the tests.

e.g. `got(someUrl)` cannot be mocked so it was changed to `got.get(someUrl)` permitting me to `sinon.stub(got, 'get')` in the tests.

**Review by hardillb — APPROVED** [review]

**Pull request merged**